modules/reference/pages/admin/cobbler.adoc

```diff
@@ -1,8 +1,7 @@
 [[s3-sattools-config-cobbler]]
 = Cobbler
 
-On the menu:Admin[Manager Configuration > Cobbler] page you can run the Cobbler synchronization by clicking btn:[Update].
+The menu:Admin[Manager Configuration > Cobbler] page allows you to run a Cobbler synchronization.
+You can repair or rebuild the contents of [path]``/srv/tftpboot`` or [path]``/srv/www/cobbler`` after a manual modification of the Cobbler setup.
 
-image::admin_configuration_cobbler.png[scaledwidth=80%]
-
-Cobbler synchronization is used to repair or rebuild the contents of [path]``/srv/tftpboot`` or [path]``/srv/www/cobbler`` when a manual modification of the cobbler setup has occurred.
+For more information on Cobbler, see xref:client-configuration:cobbler.adoc[].
```

Comment:
Wondering whether `information on` and `information about` are equally fine?  `on` is shorter ;)

Reply:
`about` is probably better, honestly. I think it meets STE guidelines.